packages/esm-bed-management-app/translations/en.json

```diff
@@ -1,49 +1,64 @@
 {
   "actions": "Actions",
   "addBed": "Add bed",
-  "addBedTag": "Create Bed Tag",
-  "addBedtype": "Add Bed Type",
-  "addBedType": "Create Bed type",
+  "addBedTag": "Create bed tag",
```

Comment:
Something we need to work on: a lot of these case-changes are probably for the best, but I believe Transifex treats them as effectively new strings meaning that these changes invalidate existing translations, even for scenarios where the capitalization is different.

Reply:
Yeah. Should we favor enforcing sentence case only for new strings? That seems like a reasonable middle ground to me. Admittedly, I'm the biggest culprit for trying to enforce visual consistency, which though nice is probably not the most important concern for translators and users of the system.

Reply:
> Should we favor enforcing sentence case only for new strings?

I'd be in favour of it. I kind of agree that sentence case looks nicer than "Button To Do The Thing".